core/vmpu/src/mpu_armv7m/vmpu_armv7m_mpu.c

```diff
@@ -136,9 +136,9 @@ uint32_t vmpu_round_up_region(uint32_t addr, uint32_t size)
 
 uint8_t vmpu_region_bits(uint32_t size)
 {
-    uint8_t bits;
+    assert(0 != size);
 
-    bits = vmpu_bits(size) - 1;
+    uint8_t bits = vmpu_bits(size) - 1;
```

Comment:
I'm particularly curious if this change is enough to make Coverity happy about this case.

Reply:
Coverity is no longer complaining about this. Nice.